### PR TITLE
10999 Handle account id sync

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#10999

*Description of changes:*
- stored account id in session
- created "navigate" function to automatically add accountid parameter
- replaced window.location.assign with new function
- fixed date/time conversions broken by Node 12
- updated breadcrumb resources to append account id
- fixed invalid URLs in unit tests
- deleted Auth API mixin (use Auth Services instead)
- app version = 3.2.3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -302,12 +300,18 @@ export default class App extends Mixins(
...
+  }
+
   /** The route breadcrumbs list. */
   private get breadcrumbs (): Array<BreadcrumbIF> {
     const crumbs: Array<BreadcrumbIF> = [
       {
         text: this.legalName || this.getNumberedEntityName,
-        href: `${sessionStorage.getItem('DASHBOARD_URL')}${this.getEntityIdentifier}`
+        href: `${sessionStorage.getItem('DASHBOARD_URL')}${this.getEntityIdentifier}/${this.getParams()}`
       },
```
Comment (severinbeauvais) [inline src/App.vue:315]: I'm thinking of moving this breadcrumb to BreadCrumbResources.ts to get rid of the duplicate `getParams()` above, but on the other hand I'd have to pass in legalName, getNumberedEntityName and getEntityIdentifier. Thoughts?

Comment (cameron-freshworks) [reply src/App.vue:315]: Fine with whatever, should work as long as our resources are localized.
Is it easy to bind Getters in a resource file?

Comment (severinbeauvais) [reply src/App.vue:315]: It's possible, yes. For example:
```
import { getVuexStore } from '@/store'
const store = getVuexStore()
const getFilingType = store.getters.getFilingType as FilingTypes
```

I'm not sure which I prefer. Or rather, dislike the least 😛

Comment (cameron-freshworks) [reply src/App.vue:315]: haha i hear ya. 
Your call!

**Review by cameron-freshworks — APPROVED** [review]

**Review by lekshmimallika-aot — APPROVED** [review]

File: src/App.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/App.vue]: Well, have a look at my new commit and tell me what you think.

Comment (cameron-freshworks) [reply src/App.vue]: Looks fine to me!

Comment (severinbeauvais) [reply src/App.vue]: OK. If we end up hating this code then we can change it later.

**Pull request merged**